### PR TITLE
Added chess environment

### PR DESCRIPTION
**Pull request by bovard**
image

### CODE REVIEW

File: kaggle_environments/envs/chess/chess.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    else:
+      board_repr += piece_symbols['.']
+    if chess.square_file(square) == 7:  # End of a rank
+      board_repr += "\n"
+
+  return board_repr
+
+jsonpath = path.abspath(path.join(path.dirname(__file__), "chess.json"))
+with open(jsonpath) as f:
+    specification = json.load(f)
```
Comment (lipovetz) [inline kaggle_environments/envs/chess/chess.py:103]: looks like this isn't used?

File: kaggle_environments/envs/chess/chess.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import chess
+import random
+import json
+from os import path
+
+def random_agent(obs):
+  """
+  Selects a random legal move from the board.
```
Comment (lipovetz) [inline kaggle_environments/envs/chess/chess.py:8]: Optional but docstrings and/or type hints are helpful-- consider adding them to the other functions.

**Review by lipovetz — APPROVED** [review]
LGTM

File: kaggle_environments/envs/chess/chess.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+    # Get the action (move) from the agent
+    action = active.action
+
+    # Check if the move is legal
+    try:
+        move = chess.Move.from_uci(action)
+        if move not in board_obj.legal_moves:
+            raise ValueError("Illegal move") 
+    except ValueError:
```
[the remark this answers is not in the document]
Comment (bovard) [reply kaggle_environments/envs/chess/chess.py:50]: thanks good catch!

**Pull request merged**